city_scrapers/spiders/il_investment_policy.py

```diff
@@ -0,0 +1,194 @@
+            "/li/p"
+        ):
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes="Confirm start time with agency.",
+                location={
+                    "name": "James R. Thompson Center",
+                    "address": "100 W. Randolph St., Room 16-503, Chicago, Illinois"
+                },
+                source=response.url
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            meeting_type = self._parse_meeting_type(meeting['title'])
+            meeting['links'] = links_map.get((meeting['start'].date(), meeting_type), [])
+
+            yield meeting
+
+    def _parse_upcoming_meetings(self, response, links_map):
+        meetings = []
+        upcoming = response.xpath("//strong[text()='2019 Board Meetings']/following::ul[1]")
+        for item in upcoming.xpath("./li"):
+            ddate_str = item.xpath("text()").get().strip()
+            ddate = datetime.strptime(ddate_str, "%m/%d/%y").date()
+            if (ddate, IlInvestmentPolicyMeetingType.IIPB) not in links_map.keys():
+                meeting = Meeting(
+                    title="IIPB Board Meeting Agenda",
+                    description="",
+                    classification=BOARD,
+                    start=datetime.combine(ddate, time(11, 00)),
+                    end=None,
+                    all_day=False,
+                    time_notes="Confirm start time with agency.",
+                    location={
+                        "name": "James R. Thompson Center",
+                        "address": "100 W. Randolph St., Room 16-503, Chicago, Illinois"
+                    },
+                    source=response.url
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+                meeting["links"] = []
+
+                meetings.append(meeting)
+        return meetings
+
+    @staticmethod
+    def _parse_title(
+        item,
+        ise_substrings=['ise', 'sudan', 'iran'],
+        ibr_substrings=['ibr', 'israel'],
+        iipb_substrings=['iipb', 'policy', 'board']
+    ):
+        """Parse or generate meeting title."""
+        title = item.xpath("text()[normalize-space(.)]").get().lstrip(' -').strip()
+        if any(substring in title.lower() for substring in ise_substrings):
+            return "Committee on Sudan and Iran Restrictions"
+        elif any(substring in title.lower() for substring in ibr_substrings):
+            return "Committee on Israel Boycott Restrictions"
+        elif any(substring in title.lower() for substring in iipb_substrings):
+            return "IIPB Board Meeting Agenda"
```

Comment:
This line should return "Investment Policy Board" instead of "Agenda" being in the title